app/javascript/src/components/Dashboard/Notes/Pane/constants.js

```diff
@@ -1,11 +1,10 @@
 import slugify from "slugify";
+import { buildSelectOption } from "utils";
 import * as yup from "yup";
 
-import { buildSelectOption } from "utils/index";
+import { CONTACTS, TAGS } from "components/Dashboard/constants";
```

Comment:
Is there a specific reason to keep `CONTACTS` literal constant within `Dashboard/constant` and not within the closest scope that is the `Contact` namespace?